pkgs/by-name/kr/kraken2/package.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub, rsync, perl, wget, gnused, findutils, libgcc, llvmPackages, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "kraken2";
+  version = "2.1.4";
+
+  src = fetchFromGitHub {
+    owner = "DerrickWood";
+    repo = pname;
+    rev = "v2.14"; # typo of current kraken2 build. version of source and software are different.
+    sha256 = "sha256-PneYibG7/u1M32f1irvBwUWyrE4gBL3evVCSj94YGQM=";
+  };
+
+  buildInputs = [
+    rsync
+    perl
+    wget
+    gnused
+    findutils
+    libgcc
+    llvmPackages.openmp
+    zlib
+  ];
+
+  postPatch = ''
+    # Assuming the install_kraken2.sh script needs to be patched or added (Linux)
+    patchShebangs ./install_kraken2.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    ./install_kraken2.sh $out/bin
+  '';
+
+  meta = with lib; {
+    description = ''
+      The second version of the Kraken taxonomic sequence classification system, 
+      using exact k-mer matches to achieve high accuracy and fast classification speeds.
+    '';
+    homepage = "https://ccb.jhu.edu/software/kraken2/";
+    license = licenses.mit;
+    platforms = platforms.unix;
+    maintainers = [ maintainers.cardealerT maintainers.A1egator];
+  };
```

```suggestion
  meta = {
    description = ''
      The second version of the Kraken taxonomic sequence classification system, 
      using exact k-mer matches to achieve high accuracy and fast classification speeds.
    '';
    homepage = "https://ccb.jhu.edu/software/kraken2/";
    changelog = "https://github.com/DerrickWood/kraken2/releases/tag/v${finalAttrs.version}";
    license = lib.licenses.mit;
    platforms = platforms.unix;
    maintainers = with lib.maintainers; [
      cardealerT
      A1egator
    ];
  };
```
`with lib;` increases eval time, which is no good, use `with` for smaller namespaces, like `lib.maintainers`